best-practice/definitions.ts

```diff
@@ -9,6 +9,56 @@ const github: readonly IBestPractice[] = [
     howToCheck: "Manual. View the repository on https://github.com",
     howToExempt: "N/A",
   },
+  {
+    name: "Branch Protection",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Enable branch protection for the default branch, ensuring changes are reviewed before being deployed.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Access",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Grant access on a team basis, rather than directly to individuals.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Collaborators",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Include at least one GitHub team as a collaborator with admin access.",
+    howToCheck: "Manual. View the repository on https://github.com",
```

Comment:
Does this link to View the repository on https://github.com/ make sense?

Reply:
Hmm possibly now. Would linking to https://github.com/orgs/guardian/repositories be a decent alternative?